client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/hooks/useDraftOutboundLines.tsx

```diff
@@ -81,14 +81,10 @@ export const useDraftOutboundLines = (
         .sort(SortUtils.byExpiryAsc);
 
       if (status === InvoiceNodeStatus.New) {
-        let placeholder = lines?.find(
+        const placeholder = lines?.find(
           ({ type }) => type === InvoiceLineNodeType.UnallocatedStock
         );
-        if (!placeholder) {
-          placeholder = draftStockOutLines.find(
-            ({ type }) => type === InvoiceLineNodeType.UnallocatedStock
-          );
-        }
```

Comment:
This was the other piece to reset to 0 on item change... when the item changed, we were stealing the placeholder line from the previous item, and reassigning it to this one, hence retaining an `issue` quantity via placeholder for that new item 🤯 

I think we only need to worry about a "previous" placeholder if we're editing an existing line (covered above)

I'm a little worried to remove it, I'm sure it was added for a reason? But I can't think of that use case... this useEffect is only triggered when item changes, or API data changes, so should be safe??

Reply:
Oh yeah - didn't touch bc placeholder line not used for prescriptions, but should remove for when we enable 😅